PageEdit/PageEdit.js

```diff
@@ -0,0 +1,41 @@
+const linkToPortfolioPage = "../Portfolio/Portfolio.html";
```

Comment:
use pascal case for constants

Reply:
Fixed, Thank you

Reply:
use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case